now.json

```diff
@@ -1,6 +1,13 @@
 {
+  "version": 2,
   "name": "nteract-site",
   "alias": "nteract.io",
+  "builds": [
+    {
+      "src": "package.json",
+      "use": "@now/node"
```

Comment:
To get this working on a personal site, I had to add the following builds to get past the stupid file browser thing that was popping up:
```
{
  "src": package.json,
  "use": @now/static-build,
  "config": {"distDir": "dist"}
}